res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1571,6 +1572,8 @@ PioneerDDJSX.panelSelectButton = function(channel, control, value, status, group
 
 PioneerDDJSX.shiftPanelSelectButton = function(channel, control, value, status, group) {
     var channelGroup;
+    PioneerDDJSX.shiftPanelSelectPressed = value ? true : false;
```

Comment:
Okay, I don't have a DDJ-SX to try this with, so I'll take your word for it. I think this will be ready for merge after addressing the comments about the loop related buttons below.